test/functional/p2p_invalid_messages.py

```diff
@@ -283,6 +285,25 @@ def test_invalid_pow_headers_msg(self):
             peer.send_message(msg_headers([blockheader]))
             peer.wait_for_disconnect()
 
+    def test_noncontinuous_headers_msg(self):
+        self.log.info("Test headers message with non-continuous headers sequence is logged as misbehaving")
+        block_hashes = self.generate(self.nodes[0], 10)
+        block_headers = []
```

Comment:
absolutely don't have to take, I just like using this pythonism

```suggestion
        block_headers = [from_hex(CBlockHeader(), self.nodes[0].getblockheader(block_hash, False) for block_hash in block_hashes]
```

Reply:
Nice indeed, will take it if I have to retouch.

Reply:
Agreeing with @instagibbs, the inline for loop init feels more idiomatic, but certainly not a blocker